api/schema/imageBuilder.yaml

```diff
@@ -671,7 +671,6 @@ paths:
           content:
             application/json:
               schema:
-                type: array
```

Comment:
@mgold1234 Added the change directly in the image-builder repo https://github.com/osbuild/image-builder/pull/1165

The check for manual changes in the API should be hopefully passing after running `npm run api:generate`

Reply:
I made the change prior to her PR being merged, I'll rerun the api-gen to confirm the changes though.

Reply:
hmm I'm not seeing the API-gen update as per @mgold1234's changes on the above ticket.
<img width="821" alt="Screenshot 2024-05-01 at 9 02 41 PM" src="https://github.com/osbuild/image-builder-frontend/assets/38083295/9016b269-d8fe-4489-ab62-5589160fd4e9">

It keeps attempting to append the above array. 
Maybe I need some permission that I don't understand? Perhaps for the bash script?

Reply:
maybe you should update the main branch and rebase this one?

Reply:
after this pr at the backend side getting merge, it will solve that problem in the frontend side https://github.com/osbuild/image-builder/pull/1168

Reply:
Thank you for looking into the API problem @mgold1234! :heart: